docs/tsconfig.json

```diff
@@ -1,39 +1,11 @@
 {
```

Comment:
The project doesn't compile for me right now, the tsconfig is missing the compiler option: "allowArbitraryExtensions" which should be `true`

Reply:
That is weird. It is building for me, deployed from my repo, and even here in this PR there's a preview deployment on Netlify.

Reply:
Specifically the netlify build only runs on the initial PR when it's uploaded, so if further commits change things I don't think it redeploys again...